vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
Right, yeah, I guess we would have to put the path fix in the `ui_components` railtie then.  Then we make sure that everyone's using the `react-rails` version.